### ISSUE TITLE
onnxruntime::ProviderLibrary::Get [ONNXRuntimeError] : 1 : FAIL : LoadLibrary failed with error 126

### ISSUE THREAD

**Issue by ColainCYY** [microsoft/onnxruntime#15035]
### Describe the issue

Hi,
My Java api with OpenVINO EPs has failed. The error message is：

> ai.onnxruntime.OrtException: Error code - ORT_RUNTIME_EXCEPTION - message: E:\onnxruntime\onnxruntime\core\session\provider_bridge_ort.cc:1069 onnxruntime::ProviderLibrary::Get [ONNXRuntimeError] : 1 : FAIL : LoadLibrary failed with error 126 "" when trying to load "E:\onnxruntime\build\Windows\RelWithDebInfo\RelWithDebInfo\onnxruntime_providers_openvino.dll"

I run on Windows / CPU.

I'm Sure the .dll file exists in the aim path. The error occured at the java-line "opts.addOpenVINO("");". As my code can run successfully without the line, I thought the error is not related to the path itself. That is to say, onnxruntime_providers_openvino.dll has been found, but could not be loaded.

Is it caused by incompatible versions?

### To reproduce

I build onnxruntime-openvino-java by：
`.\build.bat --config RelWithDebInfo --use_openvino CPU_FP32 --build_shared_lib --cmake_generator "Visual Studio 16 2019" --skip_tests --build_java`

I set the native path by：
`System.setProperty("onnxruntime.native.path", "E:\\onnxruntime\\build\\Windows\\RelWithDebInfo\\RelWithDebInfo");`

### Platform

Windows

### OS Version

Windows 10

### ONNX Runtime Installation

Built from Source

### ONNX Runtime Version or Commit ID

1.41.0

### ONNX Runtime API

Java

### Architecture

X64

### Execution Provider

OpenVINO

### Execution Provider Library Version

2021.4.752

**Comment by ColainCYY** [comment]
I get onnexruntime source code from https://github.com/intel/onnxruntime/releases
other than https://github.com/microsoft/onnxruntime/.
is it OK?

**Comment by skottmckay** [comment]
Have you installed OpenVINO and made sure those libraries are in the path and loadable?

https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html

**Comment by ColainCYY** [comment]
> Have you installed OpenVINO and made sure those libraries are in the path and loadable?
> 
> https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html

Thanks for your reply. I have installed OpenVINO and excute "setupvars.bat" before I built onnxruntime. But how to set these environment paras for my java demos?

**Comment by yuslepukhin** [comment]
In the command line environment, you can simply do `where` for each of the DLLs of interest and see if it can be found.

**Comment by yuslepukhin** [comment]
I do not think you need to pass anything to java demos. After onnxruntime.dll is loaded, the rest is really up to the OS. Take a look at [library search order](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order#standard-search-order-for-desktop-applications).

**Comment by Craigacp** [comment]
If you built the jar you shouldn't need to set "onnxruntime.native.path" as it will unpack it from the jar automatically, and if you do set it you need to set it before any ONNX Runtime code is initialized as the first thing it does is configure the native libraries.
